src/structs/model.rs

```diff
@@ -571,3 +571,5 @@ impl PartialEq for Model {
             && self.hetero_chains == other.hetero_chains
     }
 }
+
+impl Eq for Model {}
```

Comment:
As Modelcontains multiple f64 values it cannot implement Eq as f64 does not implement Eq.